docs/dev/debugging.md

```diff
@@ -1,44 +1,66 @@
-# Debugging vs Code plugin and the Language Server
+# Debugging VSCode plugin and the language server
 
-**NOTE:** the information here is mostly obsolete
+## Prerequisites
 
-Install [LLDB](https://lldb.llvm.org/) and the [LLDB Extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).
+- Install [LLDB](https://lldb.llvm.org/) and the [LLDB Extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).
```

Comment:
Does anyone have luck using this extension on windows? It doesn't really work well for me.

Reply:
What do you use instead?

Reply:
In Windows I use https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools

Reply:
It's author says it works: https://www.reddit.com/r/rust/comments/f1qsx9/debugging_rust_in_vscode_in_2020/fh8f1gx/.